### ISSUE TITLE
-p switch advertised by 'doom help install' but not recognized by 'doom install'

### ISSUE THREAD

**Issue by ameuret** [hlissner/doom-emacs#4643]
**What did you expect to happen?**

I expected this command:

```
.emacs.d/bin/doom -p ~/.config/doom install 
```

to install a new config in `~/.config/doom`

**What actually happened?**

```
x There was an unexpected error
  Message:
  Data: (user-error . "Unrecognized switch \"-p\"")
  Backtrace:
    (signal user-error ("Unrecognized switch \"-p\""))
```

**Additional details:**

`doom help install` literally states:

```

The location of DOOMDIR can be changed with the -p option, or by setting the
DOOMDIR environment variable. e.g.

  doom -p ~/.config/doom install
  DOOMDIR=~/.config/doom doom install

```

**Comment by destroyer449** [comment]
Doom already supports having the DOOMDIR at ~/.config/doom. So while this is assuredly a problem, it is not something that really matters in this case.

**Comment by ameuret** [comment]
I don’t get your point.

**Comment by destroyer449** [comment]
Sorry, the way I wrote my previous comment was really stupid. I was trying to say that if you just put your doom config in ~/.config/doom, it will automatically consider it your DOOMDIR if ~/.doom.d does not exist. You do not need to use the -p option if you are only trying to get your config into ~/.config/doom. If you want to use any other folder though, you will need the -p option (which is where the problem arises).

**Comment by hlissner** [comment]
I've removed the mention of `-p` in the docs in 6b173c5. The option was removed some time ago, but may come back after [the CLI rewrite](https://github.com/hlissner/doom-emacs/issues/4273). 

That said, setting the `DOOMDIR` envvar will always work and is the canonical way to do it now.

Side note: Doom looks for your config in `~/.config/doom` and `~/.doom.d`, in that order, so you don't need to explicitly set `DOOMDIR` to `~/.config.doom`.
